justfile

```diff
@@ -1,16 +1,16 @@
 # vim: set ft=make :
 # See https://just.systems/man/en
-
 #
 # Config
 #
 
 registry := "cr.l5d.io/linkerd"
```

Comment:
Hmm, I had a similar conversation with @alpeb on the PR that introduced the just file. I wanted to be consistent with what we're doing in the linkerd2 repo; when we build images locally they get tagged with `cr.l5d.io`. Since we don't actually pull any of the images it shouldn't impact the scarf stats.

We could make the registry configurable though or simply use `ghcr` everywhere. I'm not sure what we'd prefer here.

Reply:
if it's really never being used we could use a randomized registry like we are here https://github.com/linkerd/linkerd2/blob/82d85923a952c4446b3765919633cf20798b42b5/justfile#L3-L7

Reply:
Made the change. Bit of a caveat: the test set-up for proxy-init is really simple, image tags are hardcoded in the manifests (e.g [here](https://github.com/linkerd/linkerd2-proxy-init/blob/master/integration_test/iptables/iptablestest-lab.yaml#L49)). I have changed the manifests to point to `ghcr` and in CI we set `DOCKER_REGISTRY: ghcr.io/<repo-owner>`. It's a bit limiting but hopefully will be addressed when tests are re-written.